src/zadaci/zadatak1/Classroom.java

```diff
@@ -0,0 +1,31 @@
+package zadaci.zadatak1;
+
+import java.util.ArrayList;
+
+public class Classroom {
+
+    private ArrayList<zadaci.zadatak1.Students> studentsList;
```

Comment:
you might have more than 1 Students class in separate packages, `zadaci.zadatak1.Students`, create fresh repos that's the easies way to avoid this

Reply:
The class name should ideally be `Student` , because it describes one student. Students would mean there is more of them stored somehow.